### PR TITLE
Add sentry/segment configuration files, index, tests

### PR DESCRIPTION
**Pull request by euwest**
this is the first go at making sentry and segment setup into an installable
- build out sentry/segment modules with added wrapper functions
- configure karma/mocha
- add tests for tracking modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # traqing.js
 
-A simple wrapper for initialize functions of segment.io and sentry.
+A simple wrapper for segment.io and sentry that adds[ console]
```
Comment (AhoyM) [inline README.md]: Is *adds[ console]* on purpose here?

File: karma.unit.conf.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+// var karmaconfig = require('yokarmaconfig');
```
Comment (AhoyM) [inline karma.unit.conf.js]: Commented out code

File: README.md (hunk 1 above)
Comment (euwest) [inline README.md]: it is, but maybe it's weird/superfluous. the sentiment is supposed to be "adds logging(i.e. console logging) ..."

**Comment by euwest** [comment]
:bell:

File: README.md (hunk 1 above)
Comment (AhoyM) [inline README.md]: Wording is fine, I thought the spacing and punctuation looked off

**Comment by AhoyM** [comment]
:+1:

**Pull request merged**